ibm/mas_devops/roles/kmodels/templates/kmodels-controller/rbac.yml.j2

```diff
@@ -0,0 +1,37 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: km-controller-role
```

Comment:
Hi @whitfiea 
For this issue - we are waiting for research team to test because we don't know if we change this to narrow role our code will work. 

We recall that km-controller needs to have access to other namespace. 

We are going to create work item for this and fix this in future

Reply:
Hi @whitfiea ClusterRole removed